docs/addons/themes.md

```diff
@@ -3,11 +3,7 @@ title: Themes
 description: ✨ Themes for Spicetify.
 ---
 
-One of the most popular features in Spicetify is theming.
-You can customize your Spotify to your <span style={{'color': '#CB1010'}}>**heart's**</span> desire!
-**However**, this is a very cumbersome task.
-
-For this reason, the theming heroes of the Spicetify community have created a huge library of themes which can be found in the following repositories:
+The theming heroes of the Spicetify community have created a huge library of themes which can be found in the following repositories:
```

Comment:
Add some wording to say that this is not a complete exhaustive list and there may be other themes not listed.

Reply:
```suggestion
The Spicetify community has created a huge library of themes which can be found in the following repositories:
```
Let's remove the "theming heroes" 😅